exercises/house/house.py

```diff
@@ -1,6 +1,6 @@
-def verse():
+def verse(verse_text):
```

Comment:
You suggestion does not correlate with https://github.com/exercism/python/blob/master/exercises/house/house_test.py

Could you please clarify?